### PR TITLE
Mention platform support for Fuchsia OS in docs

### PR DESCRIPTION
**Pull request by marvijokhio**
Your documentation does not mention the platform support for Fuchsia OS but it is added in version 16.0 

I added it in documentation.

### CODE REVIEW

**Comment by marvijokhio** [comment]
> @googlebot I signed it!

**Comment by marvijokhio** [comment]
> > @googlebot I signed it!

/gcbrun

File: docs/getting-started/prerequisites.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ Our supported systems include:
 - **Ubuntu** (14.04, 16.04, 17.10, 18.04, 18.10)
 - **Debian** 8 (jessie) or later
 - Recent versions of **macOS** with [homebrew] (experimental)
+- **Fuchsia OS**
```
Comment (Dor1s) [inline docs/getting-started/prerequisites.md]: This documentation refers to the host system to develop / test ClusterFuzz on. The Fuchsia support that was added relatively recently is for fuzzing bots and it's not a native Fuchsia support, it uses QEMU on Linux VMs.

Comment (marvijokhio) [reply docs/getting-started/prerequisites.md]: Thanks for making it clear!

**Pull request closed without merging**